src/utils/transactions/utils.ts

```diff
@@ -32,3 +50,16 @@ export async function standardizeSafeTransaction(
     safeTxGas
   }
 }
+
+const encodeMetaTransaction = (tx: SafeBasicTransactionData): string => {
+  const data = utils.arrayify(tx.data)
+  const encoded = utils.solidityPack(
+    ['uint8', 'address', 'uint256', 'uint256', 'bytes'],
+    [tx.operation, tx.to, tx.value, data.length, data]
+  )
+  return encoded.slice(2)
+}
+
+export const encodeMultiSendData = (txs: SafeBasicTransactionData[]): string => {
```

Comment:
Do we have tests for this method`?

Reply:
There is this one (not a unit test of the method), but I will add a contract call too: https://github.com/gnosis/safe-core-sdk/pull/23/files#diff-c2f7973a9f5098cdc60417d715910ce586a7eae4a43b63cd65935b545ca99791R214

Reply:
Added the multisend with contract calls here: https://github.com/gnosis/safe-core-sdk/pull/23/files#diff-5535ed79ff3f49019cd635181eeed03ad747f8fb0d63b014008287dba9e8fe77R249